packages/fast-components-react-msft/app/app.tsx

```diff
@@ -56,6 +67,12 @@ export default class App extends React.Component<{}, {}> {
     }
 
     private handleUpdateDirection = (direction: Direction): void => {
-        // TODO: #486 change the design system direction (ltr)
+        const newDir: Direction = this.state.direction === Direction.ltr ? Direction.rtl : Direction.ltr;
```

Comment:
should this instead update `state.direction` to the value of the `direction` argument?